lib/pdn/pdn.c

```diff
@@ -150,8 +133,9 @@ int pdn_init(void)
 
 	k_sem_init(&notif_sem, 0, 1);
 
+	(void)err;
 #if defined(CONFIG_PDN_LEGACY_PCO)
```

Comment:
How about the following:
```
if (IS_ENABLED(CONFIG_PDN_LEGACY_PCO)) {
	int err = nrf_modem_at_printf("AT%XEPCO=0");

	if (err) {
		LOG_WRN("Failed to set legacy PCO mode, err %d", err);
	}
}
```
In such case there's no need to ignore the `err` variable like above.

Reply:
Thanks, after aligning the behavior when initializing manually or automatically I have more `#ifdef`s in `pdn_init()`, so I kept the `(void)err` at the top.